### DISCUSSION TITLE
sendLinkPreview issue (Waha Plus)

### DISCUSSION THREAD

**Discussion by ijro** [devlikeapro/waha#610] [Q&A] [answered]
I'm trying to use sendLinkPreview (NOWEB) without success. No image appears when sending a youtube link. I tried sendMessage directly as well (WEBJS). Am I doing something wrong?

**Comment by devlikepro** [comment]
Hi! 
In NOWEB I belive you can just `POST /api/sendText`, it'll do the magic

```json
{
  "chatId": "123@c.us",
  "text": "Hi there! https://github.com/devlikeapro/waha/discussions/610",
  "session": "noweb-files"
}
```
I've tried with this link

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
yeah, in WEBJS as well, `POST /api/sendText` works :+1: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> when sending a youtube link

Try the discussion link 

And give the youtube link you're trying to do 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by ijro** [reply]
> > when sending a youtube link
> 
> Try the discussion link
> 
> And give the youtube link you're trying to do
> 
> [![patron:PRO](https://camo.githubusercontent.com/4f70c3a9ee2450c288faea5e1a1b7ebd0ffc1571450a59676b300ffb32b32537/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

Where is the "discussion link"?
It actually worked for a few days, but now it's stopped. I tried to set a random "startTyping" and "stopTyping" before, without success.
Here's an example link: https://www.youtube.com/watch?v=GBfnuP-utNc

**Reply by ijro** [reply, marked as answer]
In fact, the full text contained a newline character (\n) that, for some reason, is messing up the link preview. I removed it and it worked. Since this character appears in some submissions, others don't, that's why the strange behavior.
